### ISSUE TITLE
top bar hides if you move the mouse within the opened top bar

### ISSUE THREAD

**Issue by eric-jahn** [tuxor1337/hidetopbar#376]
I've used for a long time, but recently (Ubuntu 22.04), if you make the top bar appear by approach the screen edge, and then you move the mouse within the newly appeared top bar space, the top bar rapidly hides and pops in and out, I think like what @semblancealgo was mentioning as well in the extension page comments.

**Comment by tuxor1337** [comment]
I can't reproduce this behavior from the description you provide.

1. Please give step-by-step instructions on how to reproduce.
2. What are your current Hide Top Bar preferences?
3. Which extensions are activated in parallel?
4. Can you reproduce the bug when no other extensions are activated?

**Comment by zorkerz** [comment]
I'm seeing a similar behavior with all other gnome and Firefox extensions disabled. For me it only occurs when Firefox is open full screen. Notably the dock auto hide exhibits the same behavior (if the gnome Ubuntu Dock extension is enabled). I thought it was a problem with Firefox but it goes away as soon as I disable Hide Top Bar.

I've been enabling and disabling other gnome extensions, occasionally the problem seems to go away temporarily but the only patter is this never occurs with Hide Top Bar disabled. In case it matters other gnome extensions I use are (Ubuntu Dock, Ubuntu App Indicators, Desktop Icons NG (DING), Unite, and Hide Keyboard Layout).

**Comment by zorkerz** [comment]
Maybe what I'm seeing is the same same as https://github.com/tuxor1337/hidetopbar/issues/372

**Comment by eric-jahn** [comment]
> For me it only occurs when Firefox is open full screen.

I just double-checked, and this is also the case for me, but for any app open full screen.  I tested by making the window of Chrome smaller than full screen, and moved Chrome into the top of the workspace, and hide-top-bar behaved as it should.  I could move the cursor left and right within the newly appearing top bar, and the top bar did not instantly vanish, as it does when the app is in  full screen/maximized mode.

**Comment by tuxor1337** [comment]
Okay, thanks for checking! I'm closing this as a duplicate of #372, please continue the discussion there.
